app/(playground)/p/[agentId]/beta-proto/artifact/server-actions.ts

```diff
@@ -6,6 +6,7 @@ import { createStreamableValue } from "ai/rsc";
 import { getCurrentMeasurementScope, isRoute06User } from "@/app/(auth)/lib";
 import { langfuseModel } from "@/lib/llm";
 import { createLogger } from "@/lib/opentelemetry";
+import { getUserId } from "@/lib/user";
```

Comment:
@koji-m I thought it would be better to use this because there is a similar fetchCurrentUser method!
https://github.com/giselles-ai/giselle/blob/53ed3f50aa76f4a1accb5d897687e34cb037dcfe/services/accounts/fetch-current-user.ts#L5

Reply:
I see, the function already exists. Fixed: https://github.com/giselles-ai/giselle/pull/195/commits/1d3559f9b36555ff7a33014167a8c3568eea9dfa